client/state/terms/actions.js

```diff
@@ -119,72 +119,122 @@ export function updateTerm( siteId, taxonomy, termId, termSlug, term ) {
 	};
 }
 
+/**
+ * Queries wp/v2 /taxonomies for a custom taxonomies rest_base. The rest_base of a taxonomy slug may not match its slug
+ * For example tags have a taxonomy slug of post_tag, but are modifiable at /wp/v2/tags/
+ *
+ * @param   {Number} siteId   Site ID
+ * @param   {String} taxonomy Taxonomy Slug
+ * @returns {Promise}         A promise that resolves to a wp/v2 taxonomy rest base string.
+ */
+const getTaxonomyRestBase = ( siteId, taxonomy ) => {
+	return new Promise( ( resolve, reject ) => {
+		switch ( taxonomy ) {
+			case 'category':
+				resolve( 'categories' );
```

Comment:
Since `register_taxonomy` can modify an existing taxonomy, would it be better to protect ourselves from a potential plugin in an AT site that could modify the `category`/`post_tag` taxonomies and change the rest base?

Reply:
I think this is going to be pretty unlikely for plugins to modify tags/categories. I'd be hesitant to add a full API request for this case.

cc @blowery @griffbrad @jsnajdr do folks have preferences?